backend/atlas_asset_pipelines/cubes/load.py

```diff
@@ -0,0 +1,107 @@
+import logging
+import time
+
+import numpy as np
+
+from backend.atlas_asset_pipelines.cubes.extract import extract_var_data
+from backend.atlas_asset_pipelines.cubes.transform import reduce_X, make_cube_index
+from backend.wmg.data.schemas.cube_schema import cube_non_indexed_dims
+from backend.wmg.data.tiledb import create_ctx
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+cube_indexed_dims_no_gene_ontology = [
+    "tissue_ontology_term_id",
+    "organism_ontology_term_id",
+]
+
+
+def load_data_into_cube(tdb_group, uri: str):
+    """
+    Load data from the concat_corpus into the queryable expression summary cube
+    """
+    ctx = create_ctx()
+    start_time = time.time()
+    logger.debug(f"Start loading big cube at : {uri}")
+
+    # extract
+    gene_ontology_term_ids = extract_var_data(tdb_group, ctx)
+    n_genes = len(gene_ontology_term_ids)
```

Comment:
Since L29-31 are extract, they should be in a load module method that is called from `job` pkg entrypoint method